src/test/java/org/takes/facets/fallback/TkFallbackTest.java

```diff
@@ -104,4 +104,23 @@ public InputStream body() {
         );
     }
 
+    /**
+     * TkFallback can fall back.
```

Comment:
@danff let's add a better description here `TkFallback can thrown an IOException when no fallback is available.`